<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="radioButton_all">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>670</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Все операции</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radiobuttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_removeOperation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить выделенную операцию</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1121</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>157</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>70</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_negative">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>690</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Только минусовые</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radiobuttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_goBack">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>660</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 153, 153);
color: rgb(254, 251, 251);</string>
   </property>
   <property name="text">
    <string>На главную</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_metals">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Все</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сортировка по металлу:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_plus">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>710</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Только Плюсовые</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radiobuttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radiobuttonGroup"/>
 </buttongroups>
</ui>
